packages/linea-ccip-gateway/src/evm-gateway/EVMGateway.ts

```diff
@@ -125,13 +125,13 @@ export class EVMGateway<T extends ProvableBlock> {
   /**
    *
    * @param address The address to fetch storage slot proofs for
-   * @param paths Each element of this array specifies a Solidity-style path derivation for a storage slot ID.
-   *              See README.md for details of the encoding.
+   * @param commands The commands to execute to compute the storage slots
+   * @param constants The constants to use in the commands
```

Comment:
the comments before seemed relevant in terms of encoding? still applicable?

Reply:
I don't see anything related to encoding in the README file, so I don't think this is relevant anymore